chroot-bin/runsommelier

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh -eu
+# Copyright (c) 2021 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+CLUTTER_BACKEND=wayland
+DISPLAY=:0
```

Comment:
other x targets have logic to select the first available DISPLAY, iirc; should copy that, or use a structure similar to `xiwi`

Reply:
commit ea8f7f9923780f99ffd4d4004ef5f499bad91920